### PR TITLE
Attempt to fix #275, #575; if accepted, supersedes #697

### LINKED ISSUE #275: Intermittent: Polybar overwrites tags area with code from template.liquid
Seemingly randomly, and more frequently since the last few commits, polybar will overwrite the tags module with code directly from template.liquid. So I'll get the end of a line like this `%{A1:$SCRIPTPATH/change_to_tag {{workspace.index}} {{tag.index}}:}` printed to the screen on the bar, but with the variables completed. What I see is `H/change_to_tag 0 6:}` overwriting tags 1 through 6. I originally wondered if there was a typo in the file, but I can't spot one. It happens intermittently, but enough to be an annoyance. It is caused by both key bindings and mouse clicks, and doesn't seem to be related to the tag index. Problem exists on community themes as well.

### LINKED ISSUE #575: Tags format issue
I'm not sure if this is a bug with LeftWM, Polybar or my template, but this is what happens:

When I switch between tags by either the keybinding or with a click, it sometimes looks like this (this is on the top left corner):
pic-select-211113-1950-04
pic-select-211113-1956-13
Sometimes it also shows something like `FFFF}` and then the tags normally.

It's not consistent but with enough attempts it eventually happens.

This is my template, I only changed a few things from the `basic_polybar` template such as spacing and the colors
```
%{A1:$SCRIPTPATH/change_to_tag {{workspace.index}} {{tag.index}}:}
%{F#000000}%{B#E60053}  {{tag.name}}  %{B-}%{F-}
%{A}
{% elsif tag.busy %}
%{A1:$SCRIPTPATH/change_to_tag {{workspace.index}} {{tag.index}}:}
%{F#0090FF}  {{tag.name}}  %{F-}
%{A}
{% else tag.visible  %}
%{A1:$SCRIPTPATH/change_to_tag {{workspace.index}} {{tag.index}}:}
%{F#FFFFFF}  {{tag.name}}  %{F-}
%{A}
{% endif %}
{% endfor %}
%{c}
{{ window_title | rstrip | lstrip | truncate: 70 }}
```

I ran `leftwm-status` to see if it has something to do with the json but they are barely different so I don't think that's the issue (I used a json prettier so they are more readable)
Before the bug:
```
{
    "window_title": "LibreWolf",
    "workspaces": [
        {
            "h": 1080,
            "index": 0,
            "layout": "MainAndVertStack",
            "tags": [
                {
                    "busy": true,
                    "focused": false,
                    "index": 0,
                    "mine": false,
                    "name": "1",
                    "visible": true
                },
                {
                    "busy": true,
                    "focused": true,
                    "index": 1,
                    "mine": true,
                    "name": "2",
                    "visible": true
                },
                {
                    "busy": false,
                    "focused": false,
                    "index": 2,
                    "mine": false,
                    "name": "3",
                    "visible": false
                },
                {
                    "busy": false,
                    "focused": false,
                    "index": 3,
                    "mine": false,
                    "name": "4",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 4,
                    "mine": false,
                    "name": "5",
                    "visible": false
                },
                {
                    "busy": false,
                    "focused": false,
                    "index": 5,
                    "mine": false,
                    "name": "6",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 6,
                    "mine": false,
                    "name": "7",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 7,
                    "mine": false,
                    "name": "8",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 8,
                    "mine": false,
                    "name": "9",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 9,
                    "mine": false,
                    "name": "10",
                    "visible": false
                }
            ],
            "w": 1920,
            "x": 1920,
            "y": 0
        },
        {
            "h": 1080,
            "index": 1,
            "layout": "MainAndVertStack",
            "tags": [
                {
                    "busy": true,
                    "focused": false,
                    "index": 0,
                    "mine": true,
                    "name": "1",
                    "visible": true
                },
                {
                    "busy": true,
                    "focused": true,
                    "index": 1,
                    "mine": false,
                    "name": "2",
                    "visible": true
                },
                {
                    "busy": false,
                    "focused": false,
                    "index": 2,
                    "mine": false,
                    "name": "3",
                    "visible": false
                },
                {
                    "busy": false,
                    "focused": false,
                    "index": 3,
                    "mine": false,
                    "name": "4",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 4,
                    "mine": false,
                    "name": "5",
                    "visible": false
                },
                {
                    "busy": false,
                    "focused": false,
                    "index": 5,
                    "mine": false,
                    "name": "6",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 6,
                    "mine": false,
                    "name": "7",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 7,
                    "mine": false,
                    "name": "8",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 8,
                    "mine": false,
                    "name": "9",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 9,
                    "mine": false,
                    "name": "10",
                    "visible": false
                }
            ],
            "w": 1920,
            "x": 0,
            "y": 0
        }
    ]
}
```
After:
```
{
    "window_title": "LibreWolf",
    "workspaces": [
        {
            "h": 1080,
            "index": 0,
            "layout": "MainAndVertStack",
            "tags": [
                {
                    "busy": true,
                    "focused": true,
                    "index": 0,
                    "mine": false,
                    "name": "1",
                    "visible": true
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 1,
                    "mine": true,
                    "name": "2",
                    "visible": true
                },
                {
                    "busy": false,
                    "focused": false,
                    "index": 2,
                    "mine": false,
                    "name": "3",
                    "visible": false
                },
                {
                    "busy": false,
                    "focused": false,
                    "index": 3,
                    "mine": false,
                    "name": "4",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 4,
                    "mine": false,
                    "name": "5",
                    "visible": false
                },
                {
                    "busy": false,
                    "focused": false,
                    "index": 5,
                    "mine": false,
                    "name": "6",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 6,
                    "mine": false,
                    "name": "7",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 7,
                    "mine": false,
                    "name": "8",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 8,
                    "mine": false,
                    "name": "9",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 9,
                    "mine": false,
                    "name": "10",
                    "visible": false
                }
            ],
            "w": 1920,
            "x": 1920,
            "y": 0
        },
        {
            "h": 1080,
            "index": 1,
            "layout": "MainAndVertStack",
            "tags": [
                {
                    "busy": true,
                    "focused": true,
                    "index": 0,
                    "mine": true,
                    "name": "1",
                    "visible": true
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 1,
                    "mine": false,
                    "name": "2",
                    "visible": true
                },
                {
                    "busy": false,
                    "focused": false,
                    "index": 2,
                    "mine": false,
                    "name": "3",
                    "visible": false
                },
                {
                    "busy": false,
                    "focused": false,
                    "index": 3,
                    "mine": false,
                    "name": "4",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 4,
                    "mine": false,
                    "name": "5",
                    "visible": false
                },
                {
                    "busy": false,
                    "focused": false,
                    "index": 5,
                    "mine": false,
                    "name": "6",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 6,
                    "mine": false,
                    "name": "7",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 7,
                    "mine": false,
                    "name": "8",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 8,
                    "mine": false,
                    "name": "9",
                    "visible": false
                },
                {
                    "busy": true,
                    "focused": false,
                    "index": 9,
                    "mine": false,
                    "name": "10",
                    "visible": false
                }
            ],
            "w": 1920,
            "x": 0,
            "y": 0
        }
    ]
}
```

It gets fixed by itself by changing the tag again but it happens somewhat frequently.

I don't think that this is actually a LeftWM bug so this is more a help request to find where the problem is and how could I fix it.
Thanks beforehand!

### PR DESCRIPTION
**Pull request by mautamu**
Hello,

@bmclean2, @franLMSP, @VentGrey would y'all mind testing this? One of the best stress tests I found is to get CPU usage in the 90%+ range with at least one browser open and a terminal window compiling something (approaching 100% RAM utilization also).

If accepted, fixes #275, fixes #575; supersedes/closes #697.

Best,
Mau

### CODE REVIEW

File: leftwm/src/bin/leftwm-state.rs [hunk 1]
```diff
@@ -172,10 +172,6 @@ fn template_handler(
         globals.insert("workspace".into(), liquid::model::Value::Object(workspace));
         //liquid only does time in utc. BUG: https://github.com/cobalt-org/liquid-rust/issues/332
         //as a workaround we are setting a time locally
```
[the remark this answers is not in the document]
Comment (mautamu) [reply leftwm/src/bin/leftwm-state.rs:174]: Ah good point, probably should also remove chrono from the deps. Will try to get to that later today.

**Comment by VentGrey** [comment]
Built on void linux and tested. Cannot really confirm a fix as I don't use polybar  however liquid responsiveness has improved a lot 😱 .  (Tested on basic lemonbar template).

**Comment by FranLMSP** [comment]
I haven't tested this branch yet (going to try it out later), but I haven't had this issue anymore on latest releases

**Comment by mautamu** [comment]
@VuiMuich Bumped.

**Pull request merged**